dotcom-rendering/src/web/components/DecideLines.tsx

```diff
@@ -0,0 +1,38 @@
+import { css } from '@emotion/react';
+import { ArticleDesign, ArticlePillar } from '@guardian/libs';
+import {
+	DottedLines,
+	StraightLines,
+} from '@guardian/source-react-components-development-kitchen';
+
+type Props = {
+	format: ArticleFormat;
+	color?: string;
+};
+
+export const DecideLines = ({ format, color }: Props) => {
+	const count = format.design === ArticleDesign.Comment ? 8 : 4;
+
+	switch (format.theme) {
+		case ArticlePillar.Sport:
+			return (
+				<DottedLines
+					cssOverrides={css`
+						display: block;
+					`}
+					count={count}
+					color={color}
+				/>
+			);
+		default:
+			return (
+				<StraightLines
+					cssOverrides={css`
+						display: block;
+					`}
```

Comment:
I believe SVG elements are block by default, so this may be redundant? The way we display these has changed slightly with the new Line components.

Reply:
As a test I tried removing this but it did end up having a visual impact, from looking in the chrome styles no `display` property was set by default and adding/removing it affected the spacing around it!